akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaRange.scala

```diff
@@ -92,6 +96,7 @@ object MediaRange {
     override def isMultipart = mediaType.isMultipart
     override def isText = mediaType.isText
     override def isVideo = mediaType.isVideo
+    override def isWildcard = mediaType.mainType == "*" && mediaType.subType == "*"
```

Comment:
I don't think that makes sense. A `MediaType` should never have "*" somewhere, so this can be always `false`.